srcpkgs/python3-openai/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python3-openai'
+pkgname=python3-openai
+version=0.25.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-requests python3-numpy python3-pandas python3-tqdm python3-pandas-stubs python3-openpyxl"
```

Comment:
I am pretty sure pandas-stubs are only necessary for development, so we don't care about that dependency.